renderlib/ViewerWindow.cpp

```diff
@@ -171,6 +174,13 @@ ViewerWindow::redraw()
     LOG_ERROR << "Failed to update selection buffer";
   }
 
+  // lazy init
+  if (!gesture.graphics.font.get()) {
+    gesture.graphics.font.reset(new Font());
+    std::string fontPath = renderlib::assetPath() + "/fonts/Arial.ttf";
+    gesture.graphics.font->load(fontPath.c_str());
+  }
+
```

Comment:
`renderer.cpp`/`pyrenderer.cpp` also have this same initialization logic. Does this need to be in both places?

Reply:
Yeah for now I have stuck this font as a member of Gesture.Graphics so each separate instance of Graphics needs to do its own font init.  I likely will move it to be owned by `renderlib` so it's more of a global singleton.